agent/structs/config_entry_jwt_provider.go

```diff
@@ -15,6 +15,12 @@ import (
 
 const (
 	DefaultClockSkewSeconds = 30
+
+	StrictDNS   ClusterDiscoveryType = "STRICT_DNS"
```

Comment:
It might be worth considering using some kind of prefix since these are exported and `structs.Static` doesn't necessarily point back to it being a string for a discovery type. 

Maybe just something like `DiscoveryType_Static`? Though that might be a little too Java-ish